client/src/components/Checkout/CheckoutSuccess.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { Typography, makeStyles, Grid, Button } from "@material-ui/core";
+import { CheckCircleOutline } from "@material-ui/icons";
+import { Link } from "react-router-dom";
+import PropTypes from "prop-types";
+
+const CheckoutSuccess = ({ amount, clientSecret }) => {
+  const classes = useStyles();
+  return (
+    <div className={classes.root}>
+      <Grid
+        container
+        className={classes.grid}
+        direction="column"
+        alignItems="center"
+        justify="space-between"
+        spacing={4}
+      >
+        <Grid item>
+          <Typography variant="h5">
+            Your payment of {amount}$CAD is completed
```

Comment:
Is it possible to choose the currency?

Reply:
Yes but i haven't thought that far yet lol